rust/stackable-cockpit/src/platform/stacklet/grafana.rs

```diff
@@ -20,16 +22,17 @@ pub(super) async fn list(
         .context(KubeSnafu)?;
 
     for service in services {
-        let conditions: Vec<Condition> = match &service.status {
-            Some(status) => status.conditions.clone().unwrap_or(vec![]),
-            None => vec![],
-        };
```

Comment:
Any particular reason we remove support for the conditions?

Reply:
For my personally think this is mostly a non-technical and more of a business/legal question.
First of the technical part anyway: `kubectl get svc -A -o json | jq '.items[]?.status` does only contain `loadBalancer`, no `conditions` for me. I have never seen a Service with .status.conditions set, but I have no 100% proof for it.

Now to the business part: I already had discussions after I added the listing of minio, grafana and co. Kind of "this implies that the tools are part of the stackable data plattform and we need to support them in case anything brakes". As a compromise I added the "Third party service" disclaimer to the "Info" column to make it clear. Hence I think we should not try to guesswork the status of the third party services, as this might imply that the services are part of our offering/support.

Regardless of this we should think about adding the "Info" column for the "Third party service" disclaimer as well as good information, such as
```
┌───────────────────────┬───────────────────────┬───────────┬──────────────────────────────┬───────────────────────────────────────────────────────────────────────────────────┐
│ Product               ┆ Name                  ┆ Namespace ┆ Endpoints                    ┆ Info                                                                              │
╞═══════════════════════╪═══════════════════════╪═══════════╪══════════════════════════════╪═══════════════════════════════════════════════════════════════════════════════════╡
│ zookeeper             ┆ simple-zk             ┆ default   ┆                              ┆                                                                                   │
├╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┤
│ opensearch-dashboards ┆ opensearch-dashboards ┆ default   ┆ http http://172.18.0.2:32682 ┆ Third party service                                                               │
│                       ┆                       ┆           ┆                              ┆ Logs view: http://172.18.0.2:32682/app/discover?security_tenant=global#/view/logs │
│                       ┆                       ┆           ┆                              ┆ Username: admin, password: adminadmin                                             │
└───────────────────────┴───────────────────────┴───────────┴──────────────────────────────┴───────────────────────────────────────────────────────────────────────────────────┘
```

Reply:
Discussed via Slack. Resolved for now.